### PR TITLE
cchsflow 0.2.3

### PR DESCRIPTION
**Pull request by wyusuf068**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
-## Features:
+## Features
+
+- Updated example in the introduction (README.MD) to reflect data that is now included in the cchsflow package.
+- Updates to vignettes to reflect latest version of package
+- Updates to variables.csv & variableDetails.csv
+
+### Variables
+
+-  <font color="green">**New**</font> [`DHHGAGE_C`](reference/DHHGAGE_cat_fun.html) - categorical age variable that groups various age categories across all CCHS cycles. Based on the continuous age variable (`DHHGAGE_cont`) that is also harmonious across all CCHS cycles.
```
Comment (DarkShadeKnigh) [inline NEWS.md]: The link is off it links to (news/reference/dh................)

File: R/age-categorical.R [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#'  7 - 35 to 39 years
+#'  
+#'  8 - 40 to 44 years
+#'  
+#'  9 - 45 to 49 years
+#'  
+#'  10 - 50 to 54 years
+#'  
+#'  11 - 55 to 59 years
+#'
```
Comment (DarkShadeKnigh) [inline R/age-categorical.R]: I suggest using a list http://r-pkgs.had.co.nz/man.html

File: Vignettes/howtoaddvariables.Rmd [hunk 3]
```diff
@@ -225,19 +225,17 @@ Pack_years_fun <-
   }
 ```
 
-More information on what each smoking variable means can be found in the Reference section.
+More information on what each smoking variable means can be found in the [Reference](reference/Pack_years_fun/html) section.
```
Comment (DarkShadeKnigh) [inline Vignettes/howtoaddvariables.Rmd]: /html is wrong link

File: Vignettes/usingcchsflow.Rmd [hunk 4]
```diff
@@ -108,12 +111,12 @@ Use the `bind_rows` function to combine datasets and create a new dataset name.
 
 ### Example 2. Transform a single variable from multiple CCHS datasets
 
-This example shows how you can transform and combine a variable across multiple CCHS cycles. The sex variable in CCHS 2001 (`DHHA_SEX`) and CCHS 2013 (`DHH_SEX`) is transformed a common variable (`DHH_SEX`) and combined into a single dataset. In `cchsflow` the CCHS 2007 variable name is used as the common variable name. See the [variableDetails vignette]() for more information.
+This example shows how you can transform and combine a variable across multiple CCHS cycles. The sex variable in CCHS 2001 (`DHHA_SEX`) and CCHS 2013 (`DHH_SEX`) is transformed a common variable (`DHH_SEX`) and combined into a single dataset. In `cchsflow` the CCHS 2007 variable name is used as the common variable name. See the [variableDetails vignette](articles/variableDetails.html) for more information.
```
Comment (DarkShadeKnigh) [inline Vignettes/usingcchsflow.Rmd]: No need for 'articles' see https://github.com/Big-Life-Lab/bllflow/blob/3a91aa80b3d7ad83f137eb86fdbc16b956dc7550/vignettes/bllflow.Rmd#L47 
for example

File: Vignettes/usingcchsflow.Rmd [hunk 5]
```diff
@@ -248,7 +251,7 @@ Derived variables in CCHS are created from existing, original variables that are
 
 Recoding `BMI` into a harmonized variable requires their underlying initial variables (height and weight). This first step of harmonizing underlying variables is completed for you, but you must have the underlying variables in your dataset. 
 
-Using the BMI example illustrated in the [derivedVariables]('~../Vignettes/derivedVariables.html') article, `RecWTable()` is able to transform BMI across multiple cycles provided that height (`HWTGHTM`) and weight (`HWTGWTK`) are specified.
+Using the BMI example illustrated in the [derivedVariables](articles/derivedVariables.html) article, `RecWTable()` is able to transform BMI across multiple cycles provided that height (`HWTGHTM`) and weight (`HWTGWTK`) are specified.
```
Comment (DarkShadeKnigh) [inline Vignettes/usingcchsflow.Rmd]: broken link

**Comment by DarkShadeKnigh** [comment]
also no need to push DSstore

File: NEWS.md (hunk 1 above)
Comment (DougManuel) [reply NEWS.md]: @wyusuf068 can you fix that?

**Pull request merged**